Library/Homebrew/formula_installer.rb

```diff
@@ -786,7 +791,7 @@ def pour
       path.extend(InstallRenamed)
       path.cp_path_sub(formula.bottle_prefix, HOMEBREW_PREFIX)
     end
-    FileUtils.rm_rf formula.bottle_prefix
+    FileUtils.rm_rf Dir["#{formula.bottle_prefix}/{etc,var}"]
```

Comment:
We have two metafiles so far, `INSTALL_RECEIPT.json` and `FORMULA.rb`. On the chance that one day there's another metafile to install, I'd be inclined to name this directory something generic, like `.brew`, so that all the metafiles can live in one suitably-named directory.

I've updated the PR using `.brew`. I'm happy to change it to whichever name you prefer.